### PR TITLE
[doc]: Explain the difference between using play-ws standalone and in a Play project

### LINKED ISSUE #7484: [doc] Play-WS migration doc confuses with new standalone version
The [Play 2.6 migration guide for WS](https://playframework.com/documentation/2.6.x/WSMigration26) incorrectly lists the dependencies for WS standalone. The dependency should be:

```
"com.typesafe.play" %% "play-ahc-ws" % "2.6.0"
```

### PR DESCRIPTION
**Pull request by marcospereira**
## Fixes

Fixes #7484.

### CODE REVIEW

File: documentation/manual/releases/release26/migration26/WSMigration26.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 # Play WS Migration Guide
 
-Play WS is now a standalone project and is available at [https://github.com/playframework/play-ws](https://github.com/playframework/play-ws). It can be added to an SBT project with:
+Play WS is now a standalone project and is available at [https://github.com/playframework/play-ws](https://github.com/playframework/play-ws). To add it to your Play projects, you can use:
+
+```scala
+libraryDependencies += ws
+```
+
+And if you want to use the cache support, add both `ws` and `wsCache` dependencies:
```
Comment (wsargent) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: Also enable cache as described in https://playframework.com/documentation/2.6.x/WsCache

File: documentation/manual/releases/release26/migration26/WSMigration26.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+
+```scala
+libraryDependencies += ws
+libraryDependencies += wsCache
```
Comment (wsargent) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: ehcache, not wsCache

Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: There is a `wsCache`, but looks like it is useless:

https://github.com/playframework/playframework/blob/adc2734b0d81c17d731551f6688b267299223ebb/framework/src/sbt-plugin/src/main/scala/play/sbt/PlayImport.scala#L72-L72

We should probably remove it, right?

File: documentation/manual/releases/release26/migration26/WSMigration26.md [hunk 3]
```diff
@@ -1,7 +1,20 @@
 <!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
 # Play WS Migration Guide
 
-Play WS is now a standalone project and is available at [https://github.com/playframework/play-ws](https://github.com/playframework/play-ws). It can be added to an SBT project with:
+Play WS is now a standalone project and is available at [https://github.com/playframework/play-ws](https://github.com/playframework/play-ws). To add it to your Play projects, you can use:
```
Comment (gmethvin) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: I would say something like "Play WS now has a [standalone version](https://github.com/playframework/play-ws) that can be used outside a Play project. If you're using it in a Play SBT project, you can still add WS by adding `libraryDependencies += ws` to your `build.sbt`. This includes the `play-ahc-ws` module, which wraps the standalone version with Play bindings, configuration, etc."

@wsargent is there anything else mentioning on the differences between the standalone and non-standalone version?

Comment (wsargent) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: Nothing that's relevant to the non-standalone users... there are some implementation details, but it won't matter to end users.

File: documentation/manual/releases/release26/migration26/WSMigration26.md (hunk 2 above)
Comment (wsargent) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: Yeah, we should remove it.  Sorry, forgot I added that.

File: documentation/manual/releases/release26/migration26/WSMigration26.md [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+Play WS now has a standalone version - [https://github.com/playframework/play-ws](https://github.com/playframework/play-ws) - that can be used outside a Play project. If you have a Play SBT project, you can still add WS by adding the following line to your `build.sbt`:
+ 
+```scala
+libraryDependencies += ws
+```
+
+This includes the `play-ahc-ws` module, which wraps the standalone version with Play Dependency Injection bindings and components, configuration and anything else that is necessary to better integrate it.
+
+
+And if you want to use the cache support, add both `ws` and [[cache dependencies|WSCache]]:
```
Comment (wsargent) [inline documentation/manual/releases/release26/migration26/WSMigration26.md]: It's not just cache dependencies -- you have to add ehcache and enable the cache explicitly.

**Comment by marcospereira** [comment]
@wsargent see it again. :-)

**Comment by marcospereira** [comment]
Backport to 2.6.x: 36c81c149fda9d0ae8c2d322b6c09cc54a248d29

**Pull request merged**